src/Validation/Validation.php

```diff
@@ -521,9 +521,11 @@ public static function localizedTime($check, $type = 'datetime', $format = null)
 
     /**
      * Boolean validation, determines if value passed is a boolean integer or true/false.
+     * To check if a value is set specifically to false or True see `inList()`.
```

Comment:
@inoas  Probably, we cannot check boolean types by using inList(), that converts boolean to string. So using backtick looks incorrect to me.